### PR TITLE
#305 - 404 page + design updates

### LINKED ISSUE #305: 404 page
We need some nice 404 page:

image

### PR DESCRIPTION
**Pull request by AleksandraKozubal**
I did 404 page along with updates and fixes in other pages. Removed some unnecessary files, added `:fresh` to environment/dev/scripts/init.sh, added shadows, removed big bubbles (from contact and company data page) and sections placement, color and size changes.

[Desktop design](https://www.figma.com/design/sdPYb47iLBwsA2FQMIu5BR/Blumilk?node-id=983-4479&t=KzjX9K6kWADLhXzz-0)
[Mobile design](https://www.figma.com/design/2QnCtqnBUqCoBnoys47J9d/MOBILE?node-id=84-4484&t=qWLFKXudoiecJBuT-0)

image
image

It should close #305.

### CODE REVIEW

File: resources/views/company-data.blade.php [hunk 1]
```diff
@@ -4,6 +4,10 @@
 @section("description", __("meta.company-data.description"))
 
 @section("content")
+    @php
+        $excludeBigBubbles = true;
+    @endphp
+
```
Comment (kamilpiech97) [inline resources/views/company-data.blade.php:10]:
```suggestion
    @php
        $excludeBigBubbles = true;
    @endphp

```

File: resources/views/company-data.blade.php [hunk 2]
```diff
@@ -4,6 +4,10 @@
 @section("description", __("meta.company-data.description"))
 
 @section("content")
+    @php
+        $excludeBigBubbles = true;
+    @endphp
```
Comment (kamilpiech97) [inline resources/views/company-data.blade.php:9]: Lest add that value into @extends('layout'), that looks ugly ;c

Comment (kamilpiech97) [reply resources/views/company-data.blade.php:9]: Default variable should be `false`.

**Review by EwelinaSkrzypacz — COMMENTED** [review]
This page title is not translated

image

File: resources/views/layout/public.blade.php [hunk 3]
```diff
@@ -25,29 +25,36 @@
 </head>
 
 <body class="font-sora flex flex-col h-screen justify-between">
+    @php $excludeBigBubbles = false; @endphp
```
Comment (kamilpiech97) [inline resources/views/layout/public.blade.php:28]:
```suggestion
    @php $excludeBigBubbles = false; @endphp
```
:eyes:

**Review by kamilpiech97 — APPROVED** [review]

**Review by EwelinaSkrzypacz — APPROVED** [review]

**Pull request merged**